clappr/clappr-ios
# Enable library_for_distribution build settings option

### Goal
Make player binary support future xcode and swift versions (minors); making use of stable ABI

### How to test
1 - Check if app is building
2 - Run tests
(you can try to import player from another project using new xcode 11.2.1)
